eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/actions/DirectionsLabelFactory.java

```diff
@@ -44,6 +44,6 @@ public DirectionsLabelFactory(Iterable<Location> locations)
     @Override
     public CharSequence create(Context context)
     {
-        return new AddressName(mLocations).value(context.getString(R.string.schedjoules_action_directions));
+        return new AddressName(mLocations).value(context.getString(R.string.schedjoules_action_directions).toUpperCase());
```

Comment:
Isn't the directions label styled like a BorderlessButton? If possible leave the up-case conversion to the styles.